.github/workflows/unit-test.yml

```diff
@@ -27,11 +27,28 @@ jobs:
       - name: Playwright install
         run: playwright install
       - name: Run unittest
-        run: pytest
+        id: pytest
+        shell: python
+        run: |
+          import os
+          import subprocess
+          import sys
+
+          result = subprocess.run(['pytest'], capture_output=True, text=True)
+          output = result.stdout
+          print(output)
+          if result.returncode > 0:
+            files = [line.strip() for line in output.splitlines() if "obtained" in line and os.path.exists(line.strip())]
+            if files:
+              print("Faulty files:", files)
+              with open(os.getenv("GITHUB_OUTPUT"), "a") as fd:
+                fd.write("pytest=" + "\n".join(files))
```

Comment:
nit. Perhaps we can rename this variable to `imgs` or something similar to indicate failed images, since `steps.pytest.outputs.pytest` is repetitive.